packages/blade/src/components/Card/CardRoot.web.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+import React from 'react';
+import type { CardRootProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import { castWebType, makeMotionTime } from '~utils';
+import { makeAccessible } from '~utils/makeAccessible';
+import { useIsMobile } from '~utils/useIsMobile';
+
+const StyledCardRoot = styled(BaseBox)<CardRootProps & { isPressed: boolean; isMobile: boolean }>(
+  ({ theme, isSelected, isFocused, scaleOnHover, isPressed, isMobile }) => {
+    const selectedColor = isSelected ? theme.colors.brand.primary[500] : 'transparent';
+    const selectedRing = `0px 0px 0px ${theme.border.width.thick}px ${selectedColor}`;
+    //  focused state
+    const focusRing = isFocused ? `, 0px 0px 0px 4px ${theme.colors.brand.primary[400]}` : '';
```

Comment:
at least isko to tokenise krlete hai kahi token me? sb jagah we are using this harcoded value 🤔

Reply:
better not to club this change together with this PR, anyways we need to solve focus ring holistically for each component.